Makefile

```diff
@@ -28,10 +28,13 @@ graphics_drm.o: graphics_drm.c minui.h graphics.h
 resources.o: resources.c minui.h
 	$(CC) -c $(CFLAGS) $(INCPATH) -o resources.o resources.c
 
+minui.pc:
+	sed "s=@LIB@=$(LIB)=g" minui.pc.in > minui.pc
```

Comment:
Not seeing renamed minui.pc.in anywhere. Maybe still on local clone but you forgot to add it.

Reply:
Oops, added the missing changes now.